### ISSUE TITLE
[Link color in alert] When using a link inside an error alert the link doesn't meet WCAG2.0 AA

### ISSUE THREAD

**Issue by line47** [18F/web-design-standards#1907]
## Description
When placing a link inside an error alert the link color contrast ratio doesn't meet WCAG2.0 AA.
*See attached screenshot. 

<img width="898" alt="screen shot 2017-04-27 at 5 06 43 pm" src="https://cloud.githubusercontent.com/assets/1449852/25530627/f2a00e82-2bf4-11e7-86d0-9342c23274df.png">

## Steps to reproduce the issue

1. Go to https://federalist.fr.cloud.gov/preview/18f/web-design-standards/develop/components/detail/alerts.html
2. Add a link into the content area of an alert error
3. See that the link to not meet WCAG2.0 AA

## Recommendation 
- Set the link color inside an alert to `initial` 
*see attached screenshot
<img width="613" alt="screen shot 2017-04-28 at 9 35 32 am" src="https://cloud.githubusercontent.com/assets/1449852/25530984/2e25ab3c-2bf6-11e7-8eba-cdbd140b4be9.png">

**Comment by shawnbot** [comment]
Thanks for this, @line47. I agree that using `color: inherit` is the fix here.

**Comment by sawyerh** [comment]
Setting it to `color: inherit` is one solution, though I'd like to propose another idea, which is to use `$color-primary-darker`, which would meet WCAG2.0 AA while still retaining the pattern of blue links. This is what we're doing in our design system, which is a variation of USWDS. 

IMO, this color difference between normal text and a link makes the link a bit more noticeable than if it were the same color as the surrounding text.

Screenshot:

![image](https://cloud.githubusercontent.com/assets/371943/26588754/35f6b4d6-4524-11e7-93ab-c5bd37541a5e.png)

**Comment by donjo** [comment]
Fixed in #1953
